src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -155,7 +161,7 @@ class BaseSidebarScreen extends Component {
                                         onSelected: () => Navigation.navigate(ROUTES.IOU_SEND),
                                     },
                                 ] : []),
-                                ...(Permissions.canUseIOU(this.props.betas) ? [
+                                ...(Permissions.canUseIOU(this.props.betas) && !this.props.network.isOffline ? [
```

Comment:
This is temporary before the CreateChatReport is implemented (as we might need to create a new chat with someone we have never chatted before too)

Reply:
Considering that the other PR is already approved and just waiting on one Web-E deploy to be merged, should we hold this PR? We would get rid of this temporary restriction and also update it to start using `getChatByParticipants`.

Reply:
Yeah I mean it i close, I just know Alberto had fun with this PR for a while, but it might be best to incorporate it now.

Reply:
I'm going to push to merge this ASAP. I will just create another PR after the chat one merges. It'll have a new, nice branch and it will likely require more changes to make sure everything matches the chat behavior. That is substantial enough to warrant a PR of its own